InterfaceFortran/tsgC2Fortran.cpp

```diff
@@ -104,30 +116,73 @@ void tsgwri_(int *id, const char *filename, int *binary){ _tsg_grid_list[*id]->w
 void tsgrea_(int *id, const char *filename){ _tsg_grid_list[*id]->read(filename); }
 
 // create
-void tsgmg_(int *id, int *dimensions, int *outputs, int *depth, int *type, int *rule, const int *anisotropic_weights, double *alpha, double *beta, const int *llimits){
+void tsgmg_(int *id, int *dimensions, int *outputs, int *depth, int *type, int *rule, int *opt_flags,
+            const int *aniso_weights, double *alpha, double *beta, const char *customRuleFilename, const int *llimits){
+    double al, be;
+    const int  *aw, *ll;
+    const char *cfn;
+
+    opt_flags[0] == 1 ? aw  = aniso_weights : aw = 0;
```

Comment:
* same as above, you should compare to `0` and not `1`, e.g., `opt_flags[0] != 0`

Reply:
This code also does not rely on C interpretation of true/false.
Do you want me to make everything be consistent with the C standard?

Reply:
* Yes, it is not in many places and it is OK if you skip a spot or two, but we should be compliant everywhere.